mmv1/products/privateca/CertificateAuthority.yaml

```diff
@@ -791,3 +804,21 @@ properties:
 
       An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass":
       "1.3kg", "count": "3" }.
+  - name: 'userDefinedAccessUrls'
```

Comment:
Terraform Provider Google can only use publicly available fields and this field as a visibility restriction on it (`CDP_AIA_TESTER`), which is probably why the test is failing (in the debug log you can see the field set in the API request, but coming back unset in the response, hence the diff).

Reply:
Makes sense, sounds good!